doc/perception_pipeline/perception_pipeline_tutorial.rst

```diff
@@ -8,13 +21,14 @@ In this section, we will walk through configuring the 3D sensors on your robot w
 * The Depth Image Occupancy Map Updater: which can take as input Depth Images (``sensor_msgs/Image``)
 
 YAML Configuration file (Point Cloud)
--------------------------------------
++++++++++++++++++++++++++++++++++++++
 
-We will have to generate a YAML configuration file for configuring the 3D sensors. Please see `this example file <https://github.com/ros-planning/panda_moveit_config/blob/master/config/sensors_kinect_pointcloud.yaml>`_ for processing point clouds, located in the `panda_moveit_config repository for Kinetic <https://github.com/ros-planning/panda_moveit_config>`_: ::
+We will have to generate a YAML configuration file for configuring the 3D sensors. Please see `this example file <https://github.com/ros-planning/panda_moveit_config/blob/master/config/sensors_kinect_pointcloud.yaml>`_ for processing point clouds, located in the `panda_moveit_config repository for Kinetic <https://github.com/ros-planning/panda_moveit_config>`_.
```

Comment:
The challenge with including that would be that it won't work out of the box as we can't directly add that.

Reply:
From a naming stand point, would `sensors_3d.yaml` be better? as more generalized.

Reply:
+1 to sensors_3d.yaml for the name

Reply:
As of now it matches the files in panda_moveit_config.
[depthmap](https://github.com/ros-planning/panda_moveit_config/blob/master/config/sensors_kinect_depthmap.yaml)
[pointcloud](https://github.com/ros-planning/panda_moveit_config/blob/master/config/sensors_kinect_pointcloud.yaml)

Also, If the name is same the user will have to delete 1 ffile and then use it. or if its all included in a single file they will have to comment out one of the plugins. Which would lead to unnecessary confusion for a newcomer.